Configuration/Objects.yaml

```diff
@@ -0,0 +1,15 @@
+Geocoder\Provider\GoogleMaps:
+  arguments:
+    1:
+      object: 'Ivory\HttpAdapter\CurlHttpAdapter'
```

Comment:
If we depends on this external lib, we should update the composer.json

Reply:
Ok

Reply:
Hmm, this is required by "willdurand/geocoder", idk if it make sense to add anoter require for "egeloen/http-adapter" here.

https://packagist.org/packages/willdurand/geocoder?q=CurlHttpAdapter&p=0#v3.3.2

Reply:
If the upstream package change the dependency, this package is broken